libs/live-config/src/featureFlags/defaultFeatures.ts

```diff
@@ -406,3 +408,14 @@ export const DEFAULT_FEATURES: Features = {
     enabled: false,
   },
 };
+
+// Firebase SDK treat JSON values as strings
+export const formatDefaultFeatures = (config: FeatureMap) =>
+  reduce(
+    config,
+    (acc, feature, featureId) => ({
+      ...acc,
+      [formatToFirebaseFeatureId(featureId)]: JSON.stringify(feature),
```

Comment:
It comes from the existing code here:
https://github.com/LedgerHQ/ledger-live/blob/b7d58b4cf3aa041b7b794d9af6f0b89bbc0df633/apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx#L22

we can not change this as they are featureflag names in firebase otherwise it would break production.

Reply:
Technically we could _duplicate_ flags in the Remote Config, with both snake_case and camelCase for some transition period... but I don't think it's worth it just to avoid that conversion function.

Reply:
No I think we should use `snake_case`, and just update the live to accept snake_case